appenders/python/README.md

```diff
@@ -1,19 +1,89 @@
-# Python custom log Handler
-Installing appender by running the setup.py script
+# Pulse Logging - Python
 
+## Installation
+Installing package via pip
+
+```bash
+pip install pulse-logging
 ```
+Alternatively you could install by running the setup script.
+
+```bash
 python setup.py install
 ```
 
-Then you can import the package
+## Usage
+#### Log Appender
+
+```python
+import logging
+from pulse import PulseHandler
+from pulse import PulseFormatter
+
+pulse_handler = PulseHandler("http://host.com:9001/v2/events/app")
+pulse_handler.setFormatter(PulseFormatter())
+pulse_handler.setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+logger.addHandler(pulse_handler)
+
+try:
+    raise ValueError
+except ValueError as e:
+    logger.exception("Bad stuff")
+```
+
+##### Logging Configuration File Example
+
+`logging.ini` file contents
```

Comment:
This would be nice to have in a file. Doesn't have to be done here, I can create an issue for it.

Reply:
Sounds good